zulipterminal/model.py

```diff
@@ -1164,15 +1164,16 @@ def _handle_update_message_flags_event(self, event: Event) -> None:
                       self.controller, -1)
 
     def formatted_local_time(self, timestamp: int,
-                             *, show_seconds: bool = False,
+                             *, show_seconds: bool,
```

Comment:
This change could be done in the previous commit, as changing show_seconds code isn't the aspect of this commit :)